_upcoming-technologies/02-ipp-scan.md

```diff
@@ -3,4 +3,9 @@ title: IPP scan (or virtual MF device) server (Scanner Application)
 ---
 The Printer Working Group (PWG) has also taken multi-function devices into account, devices which offer printer and scanner in one unit. For that they also have added IPP Scan, a driverless scanning standard for the scanners in IPP network multi-function devices.
 As IPP is already used by the operating systems for many years via CUPS, we also want to use it as scanning standard in the operating system, to easily integrate network multi-function devices and also for sandboxed packaging of scanner drivers. For this we emulate an IPP scanner, having a so-called Scanner Application, or even Printer/Scanner Applications being a driver for a whole multi-function device.
-Here we will discuss the transition from SANE-based scanning to IPP-based scanning in the operating system environment.
\ No newline at end of file
+Here we will discuss the transition from SANE-based scanning to IPP-based scanning in the operating system environment.
+
+IPPScan is an extension of the existing IPP protocol to facilitate driverless scanning. 
+Currently, to scan in linux, a user has to procure the device specific SANE driver and then use a command line or a GUI based application that communicates with that scanner using this driver. An IPPScan enabled scanner would be able to communicate and transfer standard files (pdf/jpeg) with a daemon running on the client, thus eliminating the need of scanner specific drivers. However, currently there are no such scanners enabled in the market. 
```

Comment:
For using an IPP scanner no daemon is needed. A client application can directly talk IPP with the scanner, for example using libcups for that, or for retro-fitting a SANE based client could use an IPP-Scan SANE backend.

Reply:
Till,
Thanks for clearing this up. I'll change "daemon" to it "an IPPScan SANE backend" instead.